src/main/haskell/language-kore/app/parser/Main.hs

```diff
@@ -61,3 +98,38 @@ verifyMain willChkAttr definition =
                  Left err1 -> error (printError err1)
                  Right _   -> return ()
           }
+
+
+----------------------
+-- Helper Functions --
```

Comment:
The functions below only have a single dependency each so it makes sense to move them to a more general module(they will come in handy for every Main module) but I wasn't sure where to put them.